bot/seasons/evergreen/game.py

```diff
@@ -0,0 +1,374 @@
+
+# Create aliases for complex genre names
+ALIASES = {
+    "Role-playing (rpg)": ["Role-playing", "Rpg"],
+    "Turn-based strategy (tbs)": ["Turn-based-strategy", "Tbs"],
+    "Real time strategy (rts)": ["Real-time-strategy", "Rts"],
+    "Hack and slash/beat 'em up": ["Hack-and-slash"]
+}
+
+
+class GameStatus(IntEnum):
+    """Game statuses in IGDB API."""
+
+    Released = 0
+    Alpha = 2
+    Beta = 3
+    Early = 4
+    Offline = 5
+    Cancelled = 6
+    Rumored = 7
+
+
+class AgeRatingCategories(IntEnum):
+    """IGDB API Age Rating categories IDs."""
+
+    ESRB = 1
+    PEGI = 2
+
+
+class AgeRatings(IntEnum):
+    """PEGI/ESRB ratings IGDB API IDs."""
+
+    Three = 1
+    Seven = 2
+    Twelve = 3
+    Sixteen = 4
+    Eighteen = 5
+    RP = 6
+    EC = 7
+    E = 8
+    E10 = 9
+    T = 10
+    M = 11
+    AO = 12
+
+
+class Games(Cog):
+    """Games Cog contains commands that collect data from IGDB."""
+
+    def __init__(self, bot: SeasonalBot):
+        self.bot = bot
+        self.http_session: ClientSession = bot.http_session
+
+        # Initialize genres
+        bot.loop.create_task(self._get_genres())
+
+    async def _get_genres(self) -> None:
+        """Create genres variable for games command."""
+        body = "fields name; limit 100;"
+        async with self.http_session.get(f"{BASE_URL}/genres", data=body, headers=HEADERS) as resp:
+            result = await resp.json()
+
+        genres = {genre["name"].capitalize(): genre["id"] for genre in result}
+
+        self.genres = {}
+
+        # Replace complex names with names from ALIASES
+        for genre in genres:
+            if genre in ALIASES:
+                for alias in ALIASES[genre]:
+                    self.genres[alias] = genres[genre]
+            else:
+                self.genres[genre] = genres[genre]
+
+    @group(name="games", aliases=["game"], invoke_without_command=True)
+    async def games(self, ctx: Context, genre: Optional[str] = None, amount: int = 5) -> None:
+        """
+        Get random game(s) by genre from IGDB. Use .games genres command to get all available genres.
+
+        Also support amount parameter, what max is 25 and min 1, default 5. Use quotes ("") for genres with multiple
+        words.
+        """
+        # When user didn't specified genre, send help message
+        if genre is None:
+            await ctx.send_help("games")
+            return
+
+        # Capitalize genre for check
+        genre = genre.capitalize()
+
+        # Check for amounts, max is 25 and min 1
+        if not 1 <= amount <= 25:
+            await ctx.send("Your provided amount is out of range. Our minimum is 1 and maximum 25.")
+            return
+
+        # Get games listing, if genre don't exist, show error message with possibilities.
+        try:
+            games = await self.get_games_list(amount, self.genres[genre],
+                                              offset=random.randint(0, 150))
```

Comment:
I do not find the offset, in this case, to be self-documented. Had to track down the `get_games_list` function to understand what it does. Maybe a clarification of why the offset is picked at random could be written as a comment?